### PR TITLE
feat: Expose as ESM

### PR DESCRIPTION
**Pull request by ukstv**
Move the package to be ESM and ES2020 only.

@rvagg any concerns?

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
       - name: Use Node 12
```
Comment (oed) [inline .github/workflows/main.yml:11]:
```suggestion
      - name: Use Node 16
```

**Review by oed — APPROVED** [review]
lgtm

File: .github/workflows/main.yml (hunk 1 above)
Comment (oed) [reply .github/workflows/main.yml:11]: Should we maybe test with both 14 and 16?

Comment (ukstv) [reply .github/workflows/main.yml:11]: yup

**Comment by rvagg** [comment]
sorry! late to the party, but +1!

**Pull request merged**